db/schema.rb

```diff
@@ -125,6 +126,7 @@
   end
 
   add_index "hotels", ["city"], name: "index_hotels_on_city", using: :btree
+  add_index "hotels", ["event_id"], name: "index_hotels_on_event_id", using: :btree
```

Comment:
To jest dziwne, bo w migracjach nie ma zmian, będzie trzeba to sprawdzić co się odwaliło